core/src/main/java/bisq/core/payment/payload/TransferwiseAccountPayload.java

```diff
@@ -94,6 +96,8 @@ public String getPaymentDetailsForTradePopup() {
 
     @Override
     public byte[] getAgeWitnessInputData() {
-        return super.getAgeWitnessInputData(email.getBytes(StandardCharsets.UTF_8));
+        return super.getAgeWitnessInputData(ArrayUtils.addAll(
+                email.getBytes(StandardCharsets.UTF_8),
+                getHolderName().getBytes(StandardCharsets.UTF_8)));
```

Comment:
Doesn't that break the Account age witness verification of existing accounts?

Reply:
Ah sorry, overlooked that you set an empty string which results as an empty byte array at getBytes() thus having not effect on the byte array we use for the witness.

Reply:
Maybe a comment regarding the backward compatibility and the means to enable it would be helpful.